src/plugins/blocksmith/components/audio-block-modal/index.jsx

```diff
@@ -80,6 +81,7 @@ class AudioBlockModal extends React.Component {
     this.handleSelectImage = this.handleSelectImage.bind(this);
     this.handleToggleAssetPickerModal = this.handleToggleAssetPickerModal.bind(this);
     this.handleToggleUploader = this.handleToggleUploader.bind(this);
+    this.handleChangeAside = this.handleChangeAside.bind(this);
```

Comment:
@nanetteCodes now that there are two checkboxes it might make sense to consolidate `handleChangeAside` and `handleChangeHasUpdatedDate` into a single `handleChangeCheckbox`, similar to how the article block modal already was. up to you tho. same for the others where the condition applies